docs/user-guide/expressions/functions.md

```diff
@@ -78,7 +135,7 @@ There are two ways to count unique values in `Polars`: an exact methodology an a
 --8<-- "python/user-guide/expressions/functions.py:countunique"
 ```
 
-#### Conditionals
+## Conditionals
 
 `Polars` supports if-like conditions in expression with the `when`, `then`, `otherwise` syntax. The predicate is placed in the `when` clause and when this evaluates to `true` the `then` expression is applied otherwise the `otherwise` expression is applied (row-wise).
```

Comment:
Should it be "if-else" conditions?

Also should be an s at the end of expression: "in expressions"